src/frontend/components/PublicVideoDashboard/index.tsx

```diff
@@ -34,11 +35,12 @@ const PublicVideoDashboard = ({
   const timedTextTracks = useTimedTextTrack((state) =>
     state.getTimedTextTracks(),
   );
-
+  initVideoWebsocket(video);
   if (video.live_state !== null) {
     switch (video.live_state) {
       case liveState.RUNNING:
       case liveState.PAUSED:
+      case liveState.STARTING:
```

Comment:
Yes because it depends when the live is starting. If it's the first start or after a pause. After a pause we must have the player display with the overlay explaining that the live is paused since x minutes